src/chrome/content/rules/rosyama.ru.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+www.rosyama.ru ⁴
+dev.rosyama.ru ¹
+mx.rosyama.ru ²
+
+
+¹ mismatch
+² refused
+⁴ self signed
+-->
+<ruleset name="rosyama.ru">
+	<target host="rosyama.ru" />
+
+	<target host="www.rosyama.ru" />
+
+	<rule from="^http://www\.rosyama\.ru/"
+		to="https://rosyama.ru/" />
```

Comment:
On http://www.rosyama.ru/ I get a 404, on http://rosyama.ru/ I get redirected to https://rosyama.ru/